lib/proofOfPatience.js

```diff
@@ -131,7 +131,9 @@ api.verifyToken = function(ip, did, token, callback) {
         }
 
         // decode and verify the token
-        token = jwt.verify(token, key.data, {clockTolerance: 250});
+        var clockTolerance = 
+          config.authio.proofs.proofOfPatience.clockToleranceInSecs * 1000;
```

Comment:
Is `* 1000` right? Docs for the lib say it should be in seconds, not milliseconds.